### PR TITLE
avalanchego: 1.10.5 -> 1.10.8

### PR DESCRIPTION
**Pull request by qjoly**
## Description of changes

Full changelog: https://github.com/ava-labs/avalanchego/compare/v1.10.5...v1.10.8

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/avalanchego/default.nix [hunk 1]
```diff
@@ -40,6 +40,6 @@ buildGoModule rec {
     homepage = "https://github.com/ava-labs/avalanchego";
     changelog = "https://github.com/ava-labs/avalanchego/releases/tag/v${version}";
     license = licenses.bsd3;
-    maintainers = with maintainers; [ urandom ];
+    maintainers = with maintainers; [ urandom qjoly ];
```
Comment (natsukium) [inline pkgs/applications/networking/avalanchego/default.nix:43]: This change is unrelated to the version bump, so you should split it into a separate commit.

Comment (QJoly) [reply pkgs/applications/networking/avalanchego/default.nix:43]: I've always updated the package and added my name in a single commit.

I added a second line to the commit description.

Comment (natsukium) [reply pkgs/applications/networking/avalanchego/default.nix:43]: I've always suggested separating updates from the rest.

Comment (matthiasbeyer) [reply pkgs/applications/networking/avalanchego/default.nix:43]: I agree with @natsukium - the update should be a single commit and the added maintainer should be a single commit. Both can be in one PR of course!

Comment (QJoly) [reply pkgs/applications/networking/avalanchego/default.nix:43]: I take a coffee and I update the PR

Comment (matthiasbeyer) [reply pkgs/applications/networking/avalanchego/default.nix:43]: Awesome, thanks! That reminds me that I want another cup of coffee as well! :laughing:

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
Result of `nixpkgs-review pr 250766` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>avalanchego</li>
  </ul>
</details>

**Review by natsukium — APPROVED** [review]

File: pkgs/applications/networking/avalanchego/default.nix (hunk 1 above)
Comment (natsukium) [reply pkgs/applications/networking/avalanchego/default.nix:43]: Thanks!
